src/components/Navigation.astro

```diff
@@ -1,13 +1,18 @@
 ---
-import ResponsiveToggle from './ResponsiveToggle.astro'
-import { DarkMode } from 'accessible-astro-components'
+import ResponsiveToggle from "./ResponsiveToggle.astro";
+import { DarkMode } from "accessible-astro-components";
 ---
 
-<div id="main-navigation" class="is-desktop py-8">
+<div id="main-navigation" class="is-desktop py-4">
   <div class="container">
     <a href="/" class="flex items-center gap-2 !no-underline">
-      <img src={(await import('../assets/img/logo.svg')).default} alt="Your Logo" width="47" height="37" />
-      <span class="font-bold">Accessible Astro</span>
+      <img
+        src={(await import("../assets/img/logo.webp")).default}
+        alt="Your Logo"
+        width="83"
+        height="69"
+      />
+      <!-- <span class="font-bold">Accessible Astro</span> -->
```

Comment:
We can probs get rid of this since it isnt relevant to our project

Reply:
will do

Reply:
Will do